apps/dashboard/src/components/conditions-editor/variable-select.tsx

```diff
@@ -130,53 +64,6 @@ export const VariableSelect = ({
     }
   };
 
-  const scrollToOption = (index: number) => {
```

Comment:
`scrollToOption, next, prev` moved to reusable hook that manages the keyboard navigation `useListKeyboardNavigation`